build/nuget-org.packages.config

```diff
@@ -0,0 +1,23 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I suggest moving the file to ```src\NuGet\Releases```
I think it might also be useful to have one file per release and the name of the file should be the name of the release. (e.g. "VS2015.3.packages", "dotnet.1.0.0.packages", etc.)